odoo/fields.py

```diff
@@ -1195,7 +1195,7 @@ def __get__(self, record, owner):
 
             elif self.store and record._origin and not (self.compute and self.readonly):
                 # new record with origin: fetch from origin
-                value = self.convert_to_cache(record._origin[self.name], record)
+                value = self.convert_to_cache(record._origin[self.name], record, validate=False)
```

Comment:
Makes more sense to me than the original proposal.  The purpose of `convert_to_cache` here is only to *adapt* the (valid) value from another record, and no validation is necessary in this context.